types/ember__test-helpers/v1/tsconfig.json

```diff
@@ -0,0 +1,28 @@
+{
+    "compilerOptions": {
+        "module": "commonjs",
+        "lib": [
+            "es6",
+            "dom"
+        ],
+        "noImplicitAny": true,
+        "noImplicitThis": true,
+        "strictNullChecks": true,
+        "strictFunctionTypes": true,
+        "baseUrl": "../../",
+        "typeRoots": [
+            "../../"
+        ],
+        "types": [],
+        "noEmit": true,
+        "forceConsistentCasingInFileNames": true,
+        "paths": {
+            "@ember/test-helpers": ["@ember/test-helpers/v1"],
```

Comment:
This is not my area of expertise, but I think you want `[ember__test-helpers/v1"]`

Reply:
@amcasey I tried it and I get
```
Older version 1 must have a "paths" entry of "ember__test-helpers/*": ["ember__test-helpers/v1/*"]
```

Reply:
What happens if you add that verbatim?

Reply:
I'm still mostly guessing, but what about
```json
            "@ember/*": ["ember__*"],
            "@ember/test-helpers": ["ember__test-helpers/v1"],
            "@ember/test-helpers/*": ["ember__test-helpers/v1/*"]
```

Seems like you still need the `@ember` path mapping.

Reply:
Once I do that, the v1 test fails because there are calls to `scrollTo` and `select`, which seem to be the new functions (i.e. not included in v1)?

Reply:
@amcasey I updated the PR and it still fails with the same error.

Reply:
Yep.  Sorry, it's over my head at this point.  The only other thing I can think of (if you can't wait for an expert to show up) is to give it exactly what it wants (in addition to the sensible path mappings you already have).

```json
            "@ember/*": ["ember__*"],
            "@ember/test-helpers": ["ember__test-helpers/v1"],
            "@ember/test-helpers/*": ["ember__test-helpers/v1/*"],
            "ember__test-helpers": ["ember__test-helpers/v1"],
            "ember__test-helpers/*": ["ember__test-helpers/v1/*"]
```

Seems like an infrastructure bug to me but, again, I don't know much about this functionality.